P/ProtocolBuffers/common.jl

```diff
@@ -0,0 +1,73 @@
+# Protobuf version numbers are weird: The version number across all languages
+# only includes the minor and patch release.
+# Each language runtime, e.g. the C++ runtime  `libprotobuf`, has its own major
+# version on top of that.
+# Thus, e.g. ProtocolBuffers, and protoc, v"22.0" matches C++ runtime v"5.22.0".
+# 
+# Cf. https://github.com/protocolbuffers/protobuf/blob/v22.0/version.json
+base_version = v"0.22.0"
+
+sources = [
+    GitSource("https://github.com/protocolbuffers/protobuf.git", "a847a8dc4ba1d99e7ba917146c84438b4de7d085"),
+    DirectorySource(joinpath(@__DIR__, "bundled")),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/protobuf
+
+atomic_patch -p1 ../patches/aarch64.patch
```

Comment:
This patch stems from upstream: https://github.com/protocolbuffers/protobuf/pull/12043/files

Reply:
Fixed